src/clear.js

```diff
@@ -0,0 +1,23 @@
+export const clearing = (e) => {
+  const click = e.target.closest('.check');
+  if (!click) return;
+
+  const dataT = parseInt(click.getAttribute('data-set'), 10);
+  const task = JSON.parse(localStorage.getItem('localItem')) || [];
+  const find = task.find((todo) => todo.index === dataT);
+  find.completed = !find.completed;
+  localStorage.setItem('localItem', JSON.stringify(task));
+};
+
+export const clear = () => {
```

Comment:
- [ ] In crud.js: I would suggest you name your variables in a way that they reveal the intention behind it. You could rename **const clear** to const **clearCompleted**

Reply:
Done! Changed clear for clearCompleted 🔧